specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountGetProperties.json

```diff
@@ -25,7 +25,13 @@
             "bypass": "AzureServices",
             "defaultAction": "Allow",
             "ipRules": [],
-            "virtualNetworkRules": []
+            "virtualNetworkRules": [],
+            "resourceAccessRules": [
+              {
+                "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+                "resourceId": "/subscriptions/a7e99807-abbf-4642-bdec-2c809a96a8bc/resourceGroups/res9407/providers/Microsoft.Synapse/workspaces/testworkspace"
```

Comment:
the {subscription-id} should be used in example, instead of real subscription ID

Reply:
The resourceId can be outside of storage account subscription, so i used a dummy guid in the example. Similar to other examples.

Reply:
Generally we should expose subscriptionID. But if this is a dummy guild, it's OK to me.